modules/netns-isolation.nix

```diff
@@ -252,46 +252,29 @@ in {
 
     services.bitcoind = {
       bind = netns.bitcoind.address;
-      rpcbind = [
-        "${netns.bitcoind.address}"
-        "127.0.0.1"
-      ];
+      rpcbind = netns.bitcoind.address;
       rpcallowip = [
-        "127.0.0.1"
-      ] ++ map (n: "${netns.${n}.address}") netns.bitcoind.availableNetns;
-      cli = let
-        inherit (config.services.bitcoind) cliBase;
-      in pkgs.writeScriptBin cliBase.name ''
-        exec netns-exec ${netns.bitcoind.netnsName} ${cliBase}/bin/${cliBase.name} "$@"
-      '';
+        bridgeIp # For operator user
+        netns.bitcoind.address
+      ] ++ map (n: netns.${n}.address) netns.bitcoind.availableNetns;
     };
     systemd.services.bitcoind-import-banlist.serviceConfig.NetworkNamespacePath = "/var/run/netns/nb-bitcoind";
 
     services.clightning.bind-addr = netns.clightning.address;
 
     services.lnd = {
       listen = netns.lnd.address;
-      rpclisten = [
-        "${netns.lnd.address}"
-        "127.0.0.1"
-      ];
-      restlisten = [
-        "${netns.lnd.address}"
-        "127.0.0.1"
-      ];
-      cliExec = mkCliExec "lnd";
```

Comment:
Is there a reason why you left lightning-loop's `cliExec`?

Reply:
For lightning-loop we would have to expose the RPC port to the main netns, which would theoretically add a new class of attack vectors. So the security argument isn't as clear-cut as for the other services.
Personally, I'd definitely favor the "main netns" approach for `lightning-loop`, now that macaroon-auth is supported. So if you agree, I'd happily add it.

Reply:
Isn't the RPC port already exposed to the main netns in the same way bitcoind's RPC port is exposed? It seems you can remove netns-exec from `mkCliExec` and the tests still pass.

Reply:
No, the RPC server listens on the netns localhost.
RPC connections to loop can only be checked with regtest because the server doesn't start before bitcoind has finished syncing.
Scenario `netnsRegtest` (with its [latest fix](https://github.com/erikarvstedt/nix-bitcoin/commit/be546abdb970815ab52514f230ce0ab242558f20)) fails when removing `cliExec`.

Reply:
Right.